### PR TITLE
Blog post "Exploring ZooKeeper-less Kafka"

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: content/blog/exploring-zookeeper-less-kafka.asciidoc [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+:icons: font
+:imagesdir: /images
+ifdef::env-github[]
+:imagesdir: ../../static/images
+endif::[]
+
+Sometimes, less is more.
+One case where that's certainly true is dependencies.
+And so it shouldn't come at a surprise that the https://kafka.apache.org/[Apache Kafka] community is eagerly awaiting the removal of the dependency to the https://zookeeper.apache.org/[ZooKeeper] service,
+which currently is used for storing Kafka metadata (e.g. about topics and partitions) as well as leader election in the cluster.
```
Comment (rk3rn3r) [inline content/blog/exploring-zookeeper-less-kafka.asciidoc:17]: When you mention the controller below, maybe also mention it here?

File: content/blog/exploring-zookeeper-less-kafka.asciidoc [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+
+Having been in the works for multiple years, ZK-less Kafka,
+also known as KRaft ("Kafka Raft metadata mode"), was recently published as an early access feature with https://blogs.apache.org/kafka/entry/what-s-new-in-apache5[Kafka 2.8].
+I.e. the perfect time to get my hands on this and get a first feeling for ZK-less Kafka myself.
+Note this post isn't meant to be a thorough evaluation or systematic testing of the new Kafka deployment mode,
+rather take it as a description of how to get started playing with ZK-less Kafka and of a few observations I made while doing so.
+
+In the world of ZK-less Kafka, there's two node roles for nodes: _controller_ and _broker_.
+Each node in the cluster can have either one or both roles ("combined nodes").
+All controller nodes elect the _active controller_ (sometimes also simply referred to as _leader_),
```
Comment (rk3rn3r) [inline content/blog/exploring-zookeeper-less-kafka.asciidoc:41]: I think controller is different from leader? Leader is the primary responsibility for a partition, while the controller is the primary for the Kafka cluster coordination? Or did that change?

Comment (gunnarmorling) [reply content/blog/exploring-zookeeper-less-kafka.asciidoc:41]: Yeah, so I was confused about this initially, too. But if you look at the different KIPs, "leader" is used there for referring to the active controller, so it's a bit ambiguous. But then the active controller also is the partition leader of the single metadata topic partition, so it's also consistent in a way :)

Comment (gunnarmorling) [reply content/blog/exploring-zookeeper-less-kafka.asciidoc]: Expanded a bit on this in the post.

**Pull request merged**